UnityProject/Assets/Scripts/Systems/Construction/Computer.cs

```diff
@@ -139,6 +139,7 @@ public void WhenDestroyed(DestructionInfo info)
 			}
 			var frame = Spawn.ServerPrefab(framePrefab, SpawnDestination.At(gameObject)).GameObject;
 			frame.GetComponent<ComputerFrame>().ServerInitFromComputer(this);
+			frame.GetComponent<Rotatable>().FaceDirection(this.GetComponent<Rotatable>().SynchroniseCurrentDirection);
```

Comment:
`frame.GetComponent<Rotatable>()?.FaceDirection(this.GetComponent<Rotatable>().SynchroniseCurrentDirection);`

preferable if you cache this as well

Reply:
well this is only accessed when destroying and making frame? 
the object gets destroyed in the next bit of code